### PR TITLE
Update post notification templates [MAILPOET-1558]

### PR DESCRIPTION
**Pull request by costasovo**

### CODE REVIEW

File: lib/Config/PopulatorData/Templates/ModularStyleStories.php [hunk 1]
```diff
@@ -150,7 +155,7 @@ private function getBody() {
                                 array (
                                   'type' => 'image',
                                   'link' => '',
-                                  'src' => $this->template_image_url . '/Modular-Logo.png',
+                                  'src' => 'http://mailpoet.info/wp-content/uploads/2017/08/Modular-Logo.png',
```
Comment (wxa) [inline lib/Config/PopulatorData/Templates/ModularStyleStories.php]: Do we really need to host all of these images on mailpoet.info if we have them on the plugin repo or locally?

Comment (costasovo) [reply lib/Config/PopulatorData/Templates/ModularStyleStories.php]: Good catch! Thank you. I've exported this template several times and I didn't notice this change.

**Comment by costasovo** [comment]
I've updated the URLs so now it should be ok.

**Comment by costasovo** [comment]
I've fixed it in Charity template. I've amended the last commit and added the fixed URLs there.

**Pull request merged**